seed/views/v3/organizations.py

```diff
@@ -1722,12 +1504,12 @@ def insert_sample_data(self, request, pk=None):
         import_record = ImportRecord.objects.create(name='Auto-Populate', super_organization=org)
 
         # Interval Data
-        filename = "example-pm-monthly-meter-usage.xlsx"
-        filepath = os.path.dirname(os.path.abspath(__file__)) + "/data/" + filename
+        filename = 'example-pm-monthly-meter-usage.xlsx'
+        filepath = f"{Path(__file__).parent.absolute()}/data/{filename}"
```

Comment:
you probably know this already, but I learned you could also use `/` and strings to create paths (no better, just thought I'd mention it)
```python
str((Path(__file__).parent / 'data' / filename).absolute())
```

Reply:
I haven't seen that, but that's a cool operator override